pmlc/target/intel_gen_ocl_spirv/pipeline.cc

```diff
@@ -98,8 +98,9 @@ void pipelineBuilder(OpPassManager &pm) {
   pm.addPass(conversion::gpu::createGpuKernelOutliningPass());
 
   // Convert GPU to comp.
+  // TODO figure out the memorySpace for Vulkan
   pm.addPass(pmlc::conversion::gpu_to_comp::createConvertGpuToCompPass(
-      comp::ExecEnvRuntime::OpenCL, /*memorySpace=*/11));
```

Comment:
I think you can move this passes to intel_gen, I think we will rename it to something like inte_gen_vulkan

Reply:
fixed